react-class-components/src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom/client';
+class CustomButton extends Component {
```

Comment:
Sorry! Looks like you didn't get my update to the instructions soon enough. Please convert this to a function component and use destructuring for the props parameter.